packages/account-v2/tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "outDir": "./dist",
         "baseUrl": "./",
         "paths": {
-            "@deriv/*": ["../*/src"]
+            "@deriv/*": ["../*/src"],
+            "@/*": ["src/*"]
```

Comment:
What does this do? Will it work with packages where we are refering to dist?

Reply:
Oh man forgot to remove this
Wanted to try to add path aliases